### DISCUSSION TITLE
Update Text of Show-InstallationProgress

### DISCUSSION THREAD

**Discussion by GubbelVogel** [PSAppDeployToolkit/PSAppDeployToolkit#768] [General]
Hello,
is it possible to update the text of the Show-InstallationProgress window at the runspace directly for the user or something similiar?
Because if I am using Show-InstallationProgress -StatusMessage at step "Pre-Installation" and then at step "Installation" the first window which I called from step "Pre-Installation" will not close until the deployment is done.
Thank you in advance.

P.S.
I am using version 3.9.2 of PSAppDeployToolkit and Powershell version 5.1 and MECM current branch
If I am using Close-InstallationProgress and calls Show-InstallationProgress again, everytime the BalloonTip will view in systemtray that the installation has started.

P.P.S.
In many cases if I only use Show-InstallationProgress -StatusMessage to change the status message it appears only a new InstallationProgress window and behind it, there is the previous one which is not closing.

**Comment by mjr4077au** [comment]
This issue has been addressed in PSADT v4
